Examples/Detectors/DD4hepDetector/include/ActsExamples/DD4hepDetector/GeoIDHooks.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "Acts/Geometry/GeometryIdentifier.hpp"
+#include "Acts/Surfaces/Surface.hpp"
+
+/// Geometry identifiers hooks to be used with DD4Hep detectors 
+/// to had some extra identifier to sensitives surfaces.
+namespace det {
+namespace geoIDHook {
```

Comment:
I think we should expand the names here to `GeometryIdentifierHooks`, and then also for the namespaces.

Reply:
Good idea.